api/users/serializers.py

```diff
@@ -1,6 +1,87 @@
 from rest_framework import serializers as ser
 from api.base.serializers import JSONAPISerializer, LinksField, Link
 from website.models import User
+from rest_framework.fields import empty, SkipField, get_attribute, CharField, MaxLengthValidator, MinLengthValidator, URLValidator, URLField
+
+
+class SocialCharField(ser.CharField):
+
+    def __init__(self, social_default, **kwargs):
+        self.allow_blank = kwargs.pop('allow_blank', False)
```

Comment:
The last line of this you can keep, but you should be able to super(SocialCharField, self).__init__(**kwargs) after it and not have to use the rest of the init method, since all of that is covered by the superclass.